README.md

```diff
@@ -156,10 +156,11 @@ The configuration can contain the following properties:
         are available:
         * **"mired"**: Using mired (more specifically _microreciprocal degree_) to calculate color temperature
         * **"kelvin"**: Using Kelvin to calculate color temperature
+        * **"percent"**: Using percent to calculate color temperature
     * `minValue` \<number\> **optional** \(Default: **50**\): Defines the minimum supported temperature in the 
-        given `unit`. The defaut is **50** mired or **20.000** Kelvin.
+        given `unit`. The defaut is **50** mired or **20.000** Kelvin or **100** percent.
     * `maxValue` \<number\> **optional** \(Default: **400**\): Defines the maximum supported temperature in the 
-        given `unit`. The fault is **400** mired or **2.500** Kelvin.
+        given `unit`. The fault is **400** mired or **2.500** Kelvin or **0** percent.
```

Comment:
minValue and maxValue seem to be the wrong way round?

Reply:
Well okay, my fault, missed the mapping of 0% -> 400 mired. Refer to the comment placed at the code handling min/max config values.

Reply:
Don't blame you at all, made the same mistake while testing it out. Was wondering what went wrong when the white light color wheel on the Home app caused the light to glow a bright yellow.

Thanks for the feedback!